### ISSUE TITLE
ideas for p2p tunnel

### ISSUE THREAD

**Issue by Aredrih** [AppliedEnergistics/Applied-Energistics-2#739]
First about p2p interaction: Actually, the p2p tunnel, which are not transferring channel, work with a "fork logic" (one seeder for infinite amount of receiver) and I wonder if that would be possible to add a "join logic" (infinte seeder for one receiver). What I have in mind is power managing because with ender io's capacitors which can output a pretty big amount of power, distributing power can be done with a p2p rf as seeder and some receiver to power a whole base (tesseract can't do better). However, to transfer the power from the generator to the capacitor, you need *one p2p by generator* because they work in a "fork logic" when a "join logic" would be more adapted to handle it (the idea isn't to make p2p OP, ok maybe a lite but there are already tesseract for that, but to "steady" it with one input, one output and not many input, one output).  

Second about p2p managing: As everyone already report, managing multiple p2p is very hard because there are no way to know whether or not a p2p is connected or not. What I suggest for that is to add a item like some glasses allowing you, when you have a link card in your hand, to see which p2p are connected to the memory card's p2p (by using highlighting for example). Also, making the light on the side of the p2p (the one which indicate if it is working) flashing when the p2p isn't connected to other p2p would simplify managing them.

Finally, a bug report (in the last DW20 pack at least): When you use p2p tunnel rf, you cannot directly output to a redstone furnace. This issue appear when you try to power any device only accepting rf (which can't output rf) directly from the p2p. It's not a so big issue since cable and cell can retrieve power from the p2p.

**tl;dr-version:** Be able to join p2p instead of just be able to fork those. Having visual effect to know more about how p2p are connected.

**Comment by yueh** [comment]
1) #31 
2) #38, #43, #395, #691, #707
3) Probably FTB is using an outdated version as usually, request an update from them.

**Comment by asgdfagdsfg** [comment]
Idea, remove this garbage and just give us back the good cables that didn't require those stupid channels!

**Comment by fdsgdfgdfh** [comment]
Just remove the shit and let people play without the brain damaged idea, of channel limitations. Removes almost 90% of your problems!
